e2e/models/onboarding/onboardingSteps.js

```diff
@@ -0,0 +1,62 @@
+import * as bridge from "../../bridge/server";
+import * as testHelpers from "../../helpers";
+
+export default class OnboardingSteps {
+  static async waitForPageToBeVisible() {
+    await testHelpers.waitForElementByText("Get started");
+  }
+
+  static async startOnboarding() {
+    await testHelpers.tapByText("Get started");
```

Comment:
Won't all these break if we change any of the wordings? Why not rely on ids like on other pages? 🤔

Reply:
I would say that test-ids are useful but don't test in the way the user would, some more infos: https://testing-library.com/docs/queries/about/#priority